src/test/scala/org/hammerlab/guacamole/commands/SomaticStandardCallerSuite.scala

```diff
@@ -238,7 +242,7 @@ class SomaticStandardCallerSuite
         ("TCGTCGA", "3M1D4M", 0)
       )
 
-    val tumorPileup = Pileup(tumorReads, "chr1", 2, simpleReference.getContig("chr1"))
+    val tumorPileup = makePileup(tumorReads)
```

Comment:
from looking at this it's not obvious to me what position the pileup will be at. Would be good to include those arguments in these places `"chr1"` and `2`